optimize/backend/src/main/java/io/camunda/optimize/service/CamundaLicenseService.java

```diff
@@ -28,4 +29,12 @@ public boolean isCamundaLicenseValid() {
   public LicenseType getCamundaLicenseType() {
     return license.getLicenseType();
   }
+
+  public boolean isCommercialCamundaLicense() {
+    return license.isCommercial();
+  }
+
+  public OffsetDateTime getCamundaLicenseExpirationDate() {
```

Comment:
Maybe `getCamundaLicenseExpiresAt` or `getCamundaLicenseExpirationDateTime` would be better here? It's not a date, and the former suggesting even keeps it more generic than any implementation

Reply:
I changed the monorepo implementation as well